platform/Stream.h

```diff
@@ -58,11 +59,12 @@ class Stream : public FileLike, private NonCopyable<Stream> {
     {
         return _file;
     }
+#endif // !MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY
+    virtual ssize_t write(const void *buffer, size_t length);
```

Comment:
Ie making these public, there's no point doing a half measure - expose the entire `FileHandle` interface. I think that's all the functions from `close` to `size`.

(Ironically, the functions it **doesn't** override from `FileHandle` are already public, eg `truncate` and `set_blocking`.)

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/6e3224290b5ec9b6a3277ef68c9073d40b23f83e..d00c27ce385afa7a96030d822586949396b5bf79).
Thanks.